sklearn/_min_dependencies.py

```diff
@@ -53,6 +53,12 @@
     for extra in extras.split(', '):
         tag_to_packages[extra].append("{}>={}".format(package, min_version))
 
+# extra dependencies
+extra_dependencies = {'numpy': 'numpy<1.20; python_version == "3.6"'}
```

Comment:
you can put numpy<1.9.3

Reply:
why?

Reply:
1.19.3 you mean? But even then, this would be too restrictive.

1.19.4 is supported on Python 3.6: https://pypi.org/project/numpy/1.19.4/#files